TweetBook/Filters/ApiKeyAuthAttribute.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+
+namespace TweetBook.Filters
+{
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method)]
+    public class ApiKeyAuthAttribute : Attribute, IAsyncActionFilter
+    {
+        private const string ApiKeyHeaderName = "ApiKey;";
+        public async Task OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
+        {
+            if(!context.HttpContext.Request.Headers.TryGetValue(ApiKeyHeaderName, out var potentialApiKey))
```

Comment:
You can use a button "Run Code Cleanup "  or command  "Ctrl+K" for so that the spacing is done automatically. After the "if" and before "(" is more better  that you have a empty space.

Reply:
Alright, thanks!